src/pages/patterns/examples/tables/TableSize.tsx

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import {
+  Body,
+  Caption,
+  Cell,
+  Head,
+  HeaderCell,
+  HeaderRow,
+  Row,
+  Table
+} from '@zendeskgarden/react-tables';
+import { LG } from '@zendeskgarden/react-typography';
+
+const StyledContainer = styled.div`
+  margin-bottom: ${p => p.theme.space.xl};
+  min-width: 500px;
+`;
+
+const Example = () => (
+  <div style={{ overflowX: 'auto' }}>
```

Comment:
If I recall, the standard in `react-components` was to avoid inline styles and prefer styled components? I'll leave that update to a different PR since I've refactored the example files and this file is no longer modified.